Preferences/Resources/Root.plist

```diff
@@ -118,7 +118,7 @@
 			<key>defaults</key>
 			<string>dev.traurige.kayoko.preferences</string>
 			<key>label</key>
-			<string>Activation Method:</string>
+			<string>Activation Method</string>
```

Comment:
Keep my wording.

Reply:
🤣 Okay. If you don't want use these modifications, just don't merge them.